core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -202,11 +204,11 @@ private[spark] object EventLoggingListener extends Logging {
   val IN_PROGRESS = ".inprogress"
   val DEFAULT_LOG_DIR = "/tmp/spark-events"
 
-  private val LOG_FILE_PERMISSIONS = new FsPermission(Integer.parseInt("770", 8).toShort)
+  val EVENT_LOG_KEY = "EVENT_LOG"
```

Comment:
I think the file name would be less of a mouthfull without this prefix. But not a big deal.